src/openqaoa-core/openqaoa/problems/vehiclerouting.py

```diff
@@ -50,20 +50,20 @@ class VRP(Problem):
     def __init__(
         self,
         G,
-        pos,
         n_vehicles,
+        pos: list = None,
```

Comment:
@vishal-ph recommends adding a default value of `None` instead because an empty list can introduce errors. I don't know what you think, let me know if you still consider changing this.

Reply:
@shahidee44, I think using immutable objects as default values is good practice. @alejomonbar, if we assign `None` as the default value I think it makes sense to update the type hinting to `Optional[list]`